scheduling-api-graphql/scheduling-api-graphql-fetchers/src/main/java/org/ow2/proactive/scheduling/api/graphql/fetchers/converter/JobInputConverter.java

```diff
@@ -124,9 +125,10 @@ public JobInputConverter(EntityManager entityManager) {
             // Index usage in queries is DB-dependant, and this "hack" applies only to HSQLDB
             predicates.add(criteriaBuilder.greaterThan(root.get("id"), 0L));
 
-            if (excludeRemoved) {
-                predicates.add(criteriaBuilder.lessThanOrEqualTo(root.get("removedTime"), 0L));
-            }
+            // remove_time support has been discontinued, thus the following setting is ignored
```

Comment:
the commented code should be removed

Reply:
I'd rather not, in case we revert this modification, I want to keep a track where it has been modified